app/src/main/res/layout/content_no_internet.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        xmlns:app="http://schemas.android.com/apk/res-auto" >
+
+    <data>
+        <variable name="title" type="com.androidtitan.culturedapp.model.newyorktimes.Article" />
+    </data>
```

Comment:
Data-binding layout files are slightly different and start with a root tag of layout followed by a data element and a view root element.

The user variable within data describes a property that may be used within this layout.